### PR TITLE
[DRFT-1209] Fix pagination overlap

### PR DESCRIPTION
**Pull request by AsToNlele**
This code fixes the Table overlap with the pagination, etc.
[DRFT-1209](https://issues.redhat.com/browse/DRFT-1209)

Before:
image
After:
image

## Secure Coding Practices Checklist Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [ ] General Coding Practices

### CODE REVIEW

**Comment by AsToNlele** [comment]
For some reason I had to update the tests and fixtures, propably a yearly thing?

File: src/SmartComponents/BaselinesTable/BaselinesToolbar/__test__/__snapshots__/BaselinesToolbar.test.js.snap [hunk 1]
```diff
@@ -202,14 +202,14 @@ exports[`jest-tests ConnectedBaselinesToolbar should render correctly 1`] = `
             Array [
               "1234",
               "beavs baseline",
-              "3 years ago",
+              "4 years ago",
```
Comment (bastilian) [inline src/SmartComponents/BaselinesTable/BaselinesToolbar/__test__/__snapshots__/BaselinesToolbar.test.js.snap:205]: I'm not sure if this should be in this PR. It doesn't affect the overlapping, or?

Comment (johnsonm325) [reply src/SmartComponents/BaselinesTable/BaselinesToolbar/__test__/__snapshots__/BaselinesToolbar.test.js.snap:205]: @bastilian It's a testing thing. We have a column in baselines that displays when a baseline was last updated. So, unless someone wants to provide me with a way to dynamically code the fixtures, we'll have to update these tests every year.

Comment (johnsonm325) [reply src/SmartComponents/BaselinesTable/BaselinesToolbar/__test__/__snapshots__/BaselinesToolbar.test.js.snap:205]: Either way, it should be included for this PR, but we shouldn't add a fix in this PR, if that makes sense.

**Review by johnsonm325 — APPROVED** [review]

**Pull request merged**